python/cudf/cudf/_lib/pylibcudf/libcudf/binaryop.pxd

```diff
@@ -19,48 +21,66 @@ cdef extern from "cudf/binaryop.hpp" namespace "cudf" nogil:
         TRUE_DIV
         FLOOR_DIV
         MOD
+        PMOD
         PYMOD
         POW
         INT_POW
+        LOG_BASE
+        ATAN2
+        SHIFT_LEFT
+        SHIFT_RIGHT
+        SHIFT_RIGHT_UNSIGNED
+        BITWISE_AND
+        BITWISE_OR
+        BITWISE_XOR
+        LOGICAL_AND
+        LOGICAL_OR
         EQUAL
         NOT_EQUAL
         LESS
         GREATER
         LESS_EQUAL
         GREATER_EQUAL
         NULL_EQUALS
-        NULL_NOT_EQUALS
-        BITWISE_AND
-        BITWISE_OR
-        BITWISE_XOR
-        LOGICAL_AND
-        LOGICAL_OR
+        NULL_MAX
```

Comment:
We're missing one operator: https://github.com/rapidsai/cudf/blob/f592e9c4bfcc2d8e887ad5f96e5167ee0ee2c73a/cpp/include/cudf/binaryop.hpp#L80
```suggestion
        NULL_NOT_EQUALS
        NULL_MAX
```